source/compliance.txt

```diff
@@ -6,48 +6,189 @@ Compliance
 
 .. default-domain:: mongodb
 
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: atlas architecture center, GDPR, ISO, PCI-DDS, FedRAMP, HIPAA
+   :description: Learn how MongoDB Atlas meets compliance and regulation controls..
+
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: onecol
 
-Intro statement
+
+This section lists the key considerations that allow you to enforce strict
+compliance controls for your deployments in {+service+}.
 
 {+service+} Features and Best Practices for Compliance
```

Comment:
Suggestion to align with a change we made on other pages (and because we're not making recommendations for this topic)
```suggestion
{+service+} Features for Compliance
```